docs/labwc-actions.5.scd

```diff
@@ -117,6 +117,16 @@ Actions are used in menus and keyboard/mouse bindings.
 	upper-left corner of the window associated with the action. Default is
 	yes.
 
+*<action name="SetDecorations" decorations="value" force_ssd="no" />*
```

Comment:
Suggest using `forceSSD="no"` to match the usual user-facing argument style.

Reply:
I had checked and found `output_name`. But you are right, camelcase that seems to be the exception.

Reply:
Whoops, that should likely be fixed then (e.g. updated in the docs, parsing `outputName` and logging an deprecation error when parsing output_name).